src/ai/backend/agent/server.py

```diff
@@ -815,11 +815,15 @@ async def push_image(
         log.info("rpc::push_image(c:{})", image_ref.canonical)
         bgtask_mgr = self.agent.background_task_manager
 
+        image_push_timeout = cast(
+            Optional[float], self.local_config["agent"]["api"]["push-timeout"]
+        )
```

Comment:
Is type casting necessary? Can’t we set the timeout value stored in local_config as Optional[float]?